### PR TITLE
Fix #17655: Style dialog allows negative values for ottava hook heigh…

### LINKED ISSUE #17655: "Hook Height Below" in Ottava style definition points in the wrong direction
### Issue type

Engraving bug

### Bug description

Reopening of #16204 as requested:
 
By changing the 'Hook Height Below' style value for ottavas, the hook can only face outwards, as the style menu only allows positive values to be input. This is newly fixable in the current master by setting a negative value in the Properties panel, but the user should be able to set a working default value for this.

### Steps to reproduce

1. Open the Style Menu/Ottavas
2. Enter a value for "Hook height below"
3. Add an ottava bassa to the score
4. The hook faces outwards

### Screenshots/Screen recordings

_No response_

### MuseScore Version

4.1

### Regression

No.

### Operating system

Windows 10

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by miiizen**
…ts consistent with inspector

Resolves: #17655

- [x] I signed the [CLA](https://musescore.org/en/cla)
- [x] The title of the PR describes the problem it addresses
- [x] Each commit's message describes its purpose and effects, and references the issue it resolves
- [x] If changes are extensive, there is a sequence of easily reviewable commits
- [x] The code in the PR follows [the coding rules](https://github.com/musescore/MuseScore/wiki/CodeGuidelines)
- [x] There are no unnecessary changes
- [x] The code compiles and runs on my machine, preferably after each commit individually
- [x] I created a unit test or vtest to verify the changes I made (if applicable)

### CODE REVIEW

**Comment by Jojo-Schmitz** [comment]
I don't think we want any of the hight and width changes

**Comment by miiizen** [comment]
These were added by Qt Creator, I believe to make space for the negative sign that can now appear in the box

**Comment by Jojo-Schmitz** [comment]
That'd be width only, not height.

**Comment by cbjeukendrup** [comment]
Qt Creator always does that, whenever making any change. I'm not sure if these changes matter anything, but I usually discard them.

**Comment by Jojo-Schmitz** [comment]
That's what I do to, and recommend in this case too

**Review by mike-spa — APPROVED** [review]

File: src/notation/view/widgets/editstyle.ui [hunk 1]
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (cbjeukendrup) [inline src/notation/view/widgets/editstyle.ui:1]: Why was this line removed? I think it's better to preserve it.

**Review by cbjeukendrup — APPROVED** [review]
